backend/src/main/java/p3/group/p3_aau_football/match/Match.java

```diff
@@ -1,44 +1,52 @@
 package p3.group.p3_aau_football.match;
 
-import java.time.LocalDateTime;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.NoSuchElementException;
-
 import org.springframework.data.annotation.Id;
 import org.springframework.data.mongodb.core.mapping.Document;
 import org.springframework.data.mongodb.core.mapping.DocumentReference;
-import p3.group.p3_aau_football.match.event.Goal;
-import p3.group.p3_aau_football.match.event.MatchEvent;
 import p3.group.p3_aau_football.role.Referee;
 import p3.group.p3_aau_football.team.Team;
 
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+import java.util.List;
+
 @Document(collection = "matches")
 public class Match {
 
     @Id
     private String id;
 
-    private String season;
-    private String competitionId;
-
     @DocumentReference
     private final Team homeTeam;
     @DocumentReference
     private final Team awayTeam;
 
-    private String venue;
+    @DocumentReference
+    private Venue venue;
     private LocalDateTime kickoff;
+    private Boolean canceled = false;
     private List<Referee> referees;
+    private LocalDate date;
 
     private int homeScore;
     private int awayScore;
-    private List<MatchEvent> matchEvents = new ArrayList<>();
-
-
+    @DocumentReference
+    private List<MatchEvent> matchEvents;
+
+    //Details
+    /*
+    (dateTime) date and time - kick-offtime splittes?
+    Venue venue
+    List<Referee> referees
+    int squadSize
+     */
```

Comment:
Der skal også være en tom constructor. Her kan du evt sætte canceled i.


```suggestion
     */
     
     public Match() {
     this.canceled = false;
     }
```